src/dispatcher/actions.ts

```diff
@@ -317,6 +317,16 @@ export enum Action {
     OnLoggedIn = "on_logged_in",
 
     /**
+     * Fired when the star button is clicked on timeline contextmenu
+     */
+    OnAddToFavourite = "on_add_to_favourite",
+
+    /**
+     * Fired when the star button is unclicked on timeline ContextMenu
+     */
+    OnRemoveFromFavourite = "on_remove_from_favourite",
+
+     /**
```

Comment:
Yeah I second @SimonBrandner - this is actions.ts in the `dispatcher` directory, this is the Flux-based dispatcher. Not the React hooks reducer dispatcher.

Reply:
@t3chguy please can you remove your change request since Usman solved this a different way and has a tick from @SimonBrandner and me?